### PR TITLE
Fixes #YUPTOO-30 - Change consumer topic from qpc to announce

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: docs/modules/ROOT/images/sequence.puml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 @startuml
```
Comment (kgaikwad) [inline docs/modules/ROOT/images/sequence.puml:1]: @patilsuraj767  - will it be possible for you to generate image using latest `sequence.puml`? Also, you would need to update yuptoo arch. diagram.

Do you want to address these changes into same PR?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Added inline comment as images need to be updated in yuptoo repository. Apart from this, rest changes look good to me :+1:

File: docs/modules/ROOT/images/sequence.puml (hunk 1 above)
Comment (patilsuraj767) [reply docs/modules/ROOT/images/sequence.puml:1]: Yes, I will update the images

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+![UML](https://www.plantuml.com/plantuml/png/VLF1Rjim3BtxAmJlkXRhslKG344FMuvhWNM7eIaCMgOJRRBaaNIwBCY_JvGRD6cHw1BalIVoyL6-OG6IeVE5EF5eVlwukx-zDNSKBJAxi30p7vyA38bUczw3j96wyw7t4Pfp224EmU8nVWNUDI0kXg874cTT3q7CUkWbnZUNN5WbADBwV1bPKcz2veEBozeLnpoSJSUo4zFnelWM1GsvnL9CR8_wdfXDIVXdG9RADUN4b53RvkBZvLNvVXvAi3R9HF4FUYuosfCBwiShhjfFrnbdzmyLqj_AXQfU2_B88AsSpMI3EbcEnEnWbLWGNAJHLTCwZsO7P7QWtAmUx6-KIlyfMbzV9TTZYfa6nZseOg4KOfPsAD20G99jjlEmBSQiNZBx4f-2HiC6o2xT2arBiPk7xvTcxFhcnjU_Gc253M4VIGnxPv7pjLsD0dp21sjrZoQTrpNk2GBhOenk_51p-bdaBMVzuQPtbJVbyxWFcx9Lctdol8mr2tPLJr5ppl3sKObTmselhCynLNO1V9NQLRyuXkcsGCyf7d1-8Dop_F1EtyXizgZUZtq7htnxRFTjdtwEBrMKUZPG9Z-XAKwdTJPTrcviVPFKG-V_0m00 "Yuptoo Processing Flow")
```
Comment (kgaikwad) [inline README.md:5]: Just an observation - In payload msg, key called `service` currently has `destination service` as value. Do you think we should update it with `qpc` for better representation?

**Review by kgaikwad — APPROVED** [review]
Legit :+1: 
Added one suggestion but I think we should not block this PR due to it.
If you think updating service key value will be helpful, we can address it separately. 
Merging this changes.

**Pull request merged**